docs/book/src/topics/addons.md

```diff
@@ -147,6 +147,138 @@ Apply kube-flannel.yml
 kubectl apply -f kube-flannel.yml
 ```
 
+## Using Azure CNI V1
+
+While following the [quick start steps in Cluster API book](https://cluster-api.sigs.k8s.io/user/quick-start.html#quick-start), Azure CNI v1 can be used in place of Calico as a container networking interface solution for your workload cluster.
```

Comment:
I think it could help to direct users to the "Deploy a CNI Solution" part of the quickstart in the link: https://cluster-api.sigs.k8s.io/user/quick-start.html#deploy-a-cni-solution

Reply:
Added the link :)